package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "js-to-do-list",
+  "version": "1.0.0",
+  "main": "script.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "description": "",
+  "dependencies": {
+    "webpack": "^5.89.0"
```

Comment:
po co webpack?

Reply:
myslalam ze webpack jest potrzebny do tego zeby zaimportowac cos z innego skryptu

"thanks to Webpack, we can split our JavaScript codebase into multiple files in a straightforward way"